mne/preprocessing/_optical_density.py

```diff
@@ -0,0 +1,46 @@
+# Authors: Robert Luke <mail@robertluke.net>
+#
+# License: BSD (3-clause)
+
+from ..io import BaseRaw
+from ..io.constants import FIFF
+from ..utils import _validate_type
+from ..io.pick import _picks_to_idx
+
+from numpy import mean, abs, log, where
```

Comment:
as a convention we always do `import numpy as np` and then `np.whatever`. It helps disambiguate things like `from math import log` from `np.log` in place without having to look at the imports at the top.

Reply:
Sorry @larsoner I should have known this from reading the contribution page.